src/components/form/Field.tsx

```diff
@@ -53,7 +55,7 @@ export function Field<T extends FieldValues, K extends InputType = "text">({
         ...(type === textarea ? {} : { type }),
         id,
         disabled: isSubmitting || disabled,
-        className: input,
+        className: `${input} ${getErrorClasses(errors, name)}`,
```

Comment:
@abbasali1994, @SovereignAndrey

I think these red borders should also be applied to other form UIs for consistency 
<img width="723" alt="image" src="https://user-images.githubusercontent.com/89639563/228793977-c66c0305-88b0-4171-904b-89f7de46130e.png">

Reply:
> I think these red borders should also be applied to other form UIs (rich text, selectors, dropzones) for consistency <img alt="image" width="723" src="https://user-images.githubusercontent.com/89639563/228793977-c66c0305-88b0-4171-904b-89f7de46130e.png">

@ap-justin, agreed that's best for consistency of UI. 
@abbasali1994, please add that into this PR. Thanks! :pray:

Reply:
okay

Reply:
Added error borders for multiselects and dropzones

Reply:
Implemented logic using aria-invalid like suggested by @ap-justin